packages/aws-cdk-lib/aws-rds/test/cluster.test.ts

```diff
@@ -268,7 +268,7 @@ describe('cluster new api', () => {
             storageType: DBClusterStorageType.AURORA_IOPT1,
             cloudwatchLogsExports: ['postgresql'],
           });
-        }).toThrow('Performance Insights must be enabled for Aurora Limitless Database.');
+        }).toThrow('DatabaseCluster initialization failed due to the following validation error(s):\n- Performance Insights must be enabled for Aurora Limitless Database\n- Performance Insights retention period must be set to at least 31 days for Aurora Limitless Database');
```

Comment:
💅🏻 nit: Might be nicer to limit the test exception to what the test is trying to test. Like for this specific test, do we really care of the other error and the prelude is included?

We could keep the existing ones and then have a single test case that checks all possible parallel rules

Reply:
The way that the validations are implemented now, the prelude will always be included. Since we were only testing for `Performance Insights must be enabled for Aurora Limitless Database`, I can change the test to include a valid retention period. However, this seems kind of contrived... since a user is unlikely to include a retention period if they did not enable performance insights.